src/KNN.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy: 0.8641304347826086\n",
+      "Classification Report:\n",
+      "              precision    recall  f1-score   support\n",
+      "\n",
+      "           0       0.81      0.86      0.83        72\n",
+      "           1       0.91      0.87      0.89       112\n",
+      "\n",
+      "    accuracy                           0.86       184\n",
+      "   macro avg       0.86      0.86      0.86       184\n",
+      "weighted avg       0.87      0.86      0.86       184\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.neighbors import KNeighborsClassifier\n",
+    "from sklearn.metrics import accuracy_score, classification_report\n",
+    "\n",
+    "# Load the dataset\n",
+    "df = pd.read_csv(r'C:\\Users\\Aksha\\Downloads\\Project Heart\\heart-resting 0 eliminated.csv')\n",
```

Comment:
1. Please, use relative path instead of absolute
2. Our data file is called heart.csv